delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/slowqueries/BlazegraphSlowQueryLogger.scala

```diff
@@ -47,15 +48,13 @@ object BlazegraphSlowQueryLogger {
         context: BlazegraphQueryContext,
         isError: Boolean,
         duration: FiniteDuration
-    ): UIO[Unit] = {
-      IOUtils.instant
-        .tapEval(_ =>
-          UIO.delay(logger.warn(s"Slow blazegraph query recorded: duration '$duration', view '${context.view}'"))
-        )
+    ): IO[Unit] = {
+      IOInstant.now
+        .flatTap(_ => logger.warn(s"Slow blazegraph query recorded: duration '$duration', view '${context.view}'"))
```

Comment:
I realise you're just doing the same as before, but it's a bit weird to flatTap when you aren't interested in the value. Maybe just the `log >> Instant.now`?